src/rabbit_control_main.erl

```diff
@@ -748,11 +761,20 @@ ensure_app_running(Node) ->
         Other -> Other
     end.
 
+notify_if_timeout(Pid, Ref, Timeout) ->
+    timer:send_after(Timeout, Pid, {Ref, {timeout, Timeout}}).
+
 call(Node, {Mod, Fun, Args}) ->
     rpc_call(Node, Mod, Fun, lists:map(fun list_to_binary_utf8/1, Args)).
 
-call(Node, {Mod, Fun, Args}, Timeout) ->
-    rpc_call(Node, Mod, Fun, lists:map(fun list_to_binary_utf8/1, Args), Timeout).
+call(Node, {Mod, Fun, Args}, InfoKeys, ToBinUtf8, Timeout) ->
```

Comment:
can we have `call/4` and `call/5`? In this way, callers to this function don't need to be seeing this `true/false` parameters all over.

With the code as it is, later, once I had forgotten what `call/5` five arguments are doing, I will have to always look at what that `true/false` flag is.